mllib/src/main/scala/org/apache/spark/mllib/clustering/KMeans.scala

```diff
@@ -503,6 +525,39 @@ object KMeans {
    *             function will be returned. (default: 1)
    * @param initializationMode The initialization algorithm. This can either be "random" or
    *                           "k-means||". (default: "k-means||")
+   * @param miniBatchFraction fraction of the input data set that should be used for
+   *                          one iteration of EM. Default value 1.0.
+   * @param seed Random seed for cluster initialization. Default is to generate seed based
+   *             on system time.
+   */
+  @Since("2.0.0")
+  def train(
```

Comment:
I don't think we want yet another overload for this parameter. It can be set via the instance methods if desired.